### ISSUE TITLE
How to reference CRS84

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/ets-ogcapi-edr10#90]
Some Abstract Tests in the Standard use the canonical URI (http://www.opengis.net/def/crs/OGC/1.3/CRS84), whereas some examples in the Standard use the short name **CRS84**.

A separate GitHub Issue (https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/324) offers a recommendation for future versions of the standard.

For the current version of the standard, since Abstract Tests are normative and examples are generally informative, we propose to configure the ETS to allow use of the canonical URI (http://www.opengis.net/def/crs/OGC/1.3/CRS84). 

The short name could then be handled in one of two ways:

* Option A: When an implementation uses the short name (**CRS84**), they would receive an FAIL result with an error message explaining the issue and advising to use the canonical URI (http://www.opengis.net/def/crs/OGC/1.3/CRS84). FAIL results prevent compliance certification.

* Option B: When an implementation uses the short name (**CRS84**), they would receive an SKIP result with a message explaining the issue and advising to use the canonical URI (http://www.opengis.net/def/crs/OGC/1.3/CRS84). SKIP results do not prevent compliance certification.

**Comment by ghobona** [comment]
Cc: @chris-little @m-burgoyne @iandruska-ibl

**Comment by chris-little** [comment]
@ghobona @m-burgoyne, @iandruska-ibl, @boyi-shangguan, @shaneMill1
I prefer Option B for V1.0.0 and Option A for V1.0.1, but want to see what the implementers think and prefer.

**Comment by boyi-shangguan** [comment]
I agree with chris that use Option B for V1.0.0 and Option A for V1.0.1.

**Comment by m-burgoyne** [comment]
I  am happy with the approach suggested by @chris-little  (Option B for V1.0.0 and Option A for V1.0.1).

**Comment by ghobona** [comment]
@chris-little Looking at the draft v1.0.1 today 2022-02-14, I see that the examples in Annex C still use:

```json
"crs" : ["CRS84"]
```

I understood your earlier response in this thread to mean that the examples would also be updated to use the value:

```
"crs" : ["http://www.opengis.net/def/crs/OGC/1.3/CRS84"]
```

Using the complete URI would also make the examples consistent with [Abstract Test 133](https://docs.ogc.org/is/19-086r4/19-086r4.html#_42873eb9-919e-c393-2bfa-9be3f197a7d2).

**Comment by ghobona** [comment]
Addressed by v1.0.1.
